From 663d0922696425b5de2ef055191c76e1b945064b Mon Sep 17 00:00:00 2001
From: Josh Wilsdon <jwilsdon@joyent.com>
Date: Tue, 23 Apr 2019 13:52:17 -0700
Subject: [PATCH] OS-7758 test-vm fails due to out of quota instead of show
 write Reviewed by: Trent Mick <trentm@gmail.com> Approved by: Trent Mick
 <trentm@gmail.com>

---
 src/vm/tests/test-update.js | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/vm/tests/test-update.js b/src/vm/tests/test-update.js
index 144c610d..dc44578b 100644
--- a/src/vm/tests/test-update.js
+++ b/src/vm/tests/test-update.js
@@ -20,7 +20,7 @@
  *
  * CDDL HEADER END
  *
- * Copyright (c) 2018, Joyent, Inc.
+ * Copyright (c) 2019, Joyent, Inc.
  *
  */
 
@@ -1286,7 +1286,7 @@ test('fill up zoneroot', function (t) {
     execFile('/usr/bin/dd', [
         'if=/dev/zero',
         'of=/zones/' + vm_uuid + '/root/zeros',
-        'bs=1M'
+        'bs=37M' // Use a value that doesn't go evenly into the 1GB VM quota.
     ], function (err, stdout, stderr) {
         var match = stderr.match(/dd: unexpected short write, wrote/);
         t.ok(match, 'expected short write'
-- 
2.21.0

